projects/optic/src/commands/oas/capture.ts

```diff
@@ -193,15 +198,21 @@ export async function captureCommand(): Promise<Command> {
                   feedback.success(`Wrote har to ${outputPath}`);
                   return fs.move(inProgressName, outputPath);
                 }
-                feedback.success(`Wrote har traffic to ${completedName}`);
-                feedback.log(
-                  chalk.gray(
-                    `\nRun ${chalk.whiteBright(
+                if (!enterPressed) {
+                  // Log next steps
+                  feedback.success(`Wrote har traffic to ${completedName}`);
+                  feedback.log(
+                    `\nRun "${chalk.bold(
                       `optic oas verify ${filePath}`
-                    )} to diff the captured traffic`
-                  )
-                );
-                return fs.move(inProgressName, completedName);
+                    )}" to diff the captured traffic`
+                  );
+                }
+                await fs.move(inProgressName, completedName);
+
+                if (enterPressed)
+                  await runVerify(filePath, { exit0: true }, config, feedback, {
```

Comment:
This is _ok_ but I think what might be better is to pull out the functions that is needed to render what you want here

I think it's just pulling out these lines https://github.com/opticdev/optic/blob/0a6c3a07b54d22d720e735437003810fe8687d2c/projects/optic/src/commands/oas/verify.ts#L186-L204 as a function that you can share across these two functions (i.e. reporting status)

Reply:
Ok I'll do that when we get to the next step -- breaking out document command again. Let's figure out the final breakup of this before refactoring it and then doing that all over again